src/include/kdbhelper.h

```diff
@@ -56,7 +56,6 @@ int elektraWriteArrayNumber (char * newName, kdb_long_long_t newIndex);
 enum elektraLookupOptions
 {
 	KDB_O_SPEC = 1 << 15,	     ///< Use the passed key as specification, instead of looking up the specification first
-	KDB_O_CREATE = 1 << 16,	     ///< Create the key if it was not found
```

Comment:
It might be worth thinking about whether one wants to keep the "gap" in the bitshifts or if  the ones "above" should be moved down.

As long as nobody used the values of the options explicitly this should be fine.